lib/src/main/kotlin/com/swmansion/starknet/deployer/StandardDeployer.kt

```diff
@@ -0,0 +1,44 @@
+package com.swmansion.starknet.deployer
```

Comment:
I would call this package universalDeployerContact or something like this. Who knows what could be used as a deployer in the future.

Reply:
You mean this implementation or the whole package with `Deployer` interface?

Reply:
package itself

Reply:
Renamed `universaldeployercontract` because it seems like it's the most popular java naming convention for multi word name.